src/devices/SenseHat/SenseHatJoystick.cs

```diff
@@ -80,20 +80,45 @@ public void Dispose()
             _i2c = null!;
         }
 
-        private JoystickState ReadState()
+        /// <summary>
+        /// Read joystick state
+        /// </summary>
+        public JoystickState GetState()
```

Comment:
I'm fine if we want to keep it like this but in general I'd rather not have two ways of doing the same thing public ina binding, unless there is a good reason to do it.